### PR TITLE
Fix: `usermod: group '...' does not exist` with custom GITEA_RUNNER_GID

### PR DESCRIPTION
**Pull request by wblew**
Second attempt to fix the GITEA_RUNNER_GID envar issue. See previous issue [#5](https://github.com/vegardit/docker-gitea-act-runner/issues/).

Tested this GITEA_RUNNER_GID envar fix without building the upstream image, as that was problematic to achieve locally.

The fixed image requires either compose: `user: root` OR `restart: always` (or `unless-stopped`). Why? The UID change in `/opt/run.sh` fails during the container's first execution, at the next `sudo` command within `/opt/run.sh`

Basically: changing the running UID's `/etc/passwd` entry causes the next sudo command to fail.

The container restart (without wiping the read/write layer) starts with the already changed `/etc/passwd`, and then continues without the `sudo` error.

See associated issue [#9](https://github.com/vegardit/docker-gitea-act-runner/issues/9) for testing method.

### CODE REVIEW

**Comment by sebthom** [comment]
Thanks!

**Comment by sebthom** [comment]
Btw. I committed a change that allows to set the GITEA_RUNNER_UID without requiring to use `user: root` in compose.
